docs/keyless-signing.md

```
@@ -0,0 +1,138 @@
+cosign login "$EXT_HOST" -u "$(oc whoami)" -p "$(oc whoami -t)"
+
+cosign verify \
+  --rekor-url "$REKOR_URL" \
+  --certificate-identity "https://kubernetes.io/namespaces/openshift-pipelines/serviceaccounts/tekton-chains-controller" \
+  --certificate-oidc-issuer "https://kubernetes.default.svc" \
+  "${EXT_HOST}/<namespace>/<image>@sha256:<digest>"
+```
+
+Expected output:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)